README.md

```diff
@@ -109,9 +109,17 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
 
 8. Install the orchestrator Helm chart:
 
-   ```console
-   helm upgrade -i orchestrator orchestrator -n orchestrator
-   ```
+    Use the following command if you use the prepackaged helm chart provided by the helm repository you previously installed:
```

Comment:
Maybe we can just leave here:
helm upgrade -i orchestrator orchestrator/orchestrator -n orchestrator
Since in step 1 user is instructed to run 
helm repo add orchestrator https://parodos-dev.github.io/orchestrator-helm-chart
No point in doing that if he'll clone the repo

We can add this one:
helm upgrade -i orchestrator orchestrator -n orchestrator
to the section:  **Manual Chart Deployment**

Reply:
Honestly, this doc is becoming too complex as is: we have `manual` vs `repo` installation and on top of that we have `with gitops` and `without gitops`. 

In case of `manual` installation, I'd move it to it's own section (`Local development`?), in case someone wants to play with the chart locally and remove any reference to it in the main installation flow. This removes an extra path.

As for the gitops/no-gitops I'm thinking of breaking them into their own documents, even if that means repeating a few steps. Thoughts?

Reply:
I agree 100%.
I don't think we need the manual installation option, anyone familiar with helm should know how to do this.
Regarding separating gitops/no-gitops I think we should leave only the gitops one.
If the user wants to install it without them, he will figure it out on his own.

Reply:
The problem with that is that the `setup.sh` script will fail now if no argocd is deployed in the kubernetes cluster. That could be amended to be optional, and then your suggestion would work well, only making the `BACKEND_SECRET` as mandatory. 

With that we can remove the local development and without gitops comments and simplify the installation. @masayag WDYT?

Reply:
What do you think about moving manual installation option to separate md file? It may still be useful for new users.

Reply:
@pkliczewski yes that's a good idea